src/main/java/org/embulk/input/jira/JiraInputPlugin.java

```diff
@@ -164,7 +164,7 @@ public ConfigDiff guess(ConfigSource config)
         Set<String> uniqueAttributes = getUniqueAttributes(issues);
         JsonArray samples = createSamples(issues, uniqueAttributes);
         Buffer sample = Buffer.copyOf(samples.toString().getBytes());
-        JsonNode columns = guessExecutor.guessParserConfig(sample, Exec.newConfigSource(), guessConfig).getObjectNode().get("parser").get("columns");
+        JsonNode columns = guessExecutor.guessParserConfig(sample, Exec.newConfigSource(), guessConfig).getObjectNode().get("columns");
```

Comment:
Not related to this PR, but please make it a habit by not introducing a variable when you don't really need it. For example, `#createGuessConfig` and  `#getGuessExecutor` could be expensive, what a waste if `#validateTaskConfig` fails.

Reply:
Agree, I also think remove some of the temp variables and inline them would make things more readable.

Reply:
To be addressed on https://github.com/treasure-data/embulk-input-jira/pull/53/files#r257135900